app/frontend/admin/courses/CourseEditor__Root.res

```diff
@@ -69,6 +77,22 @@ type state = {
   schoolStats: schoolStats,
 }
 
+let initialState = () => {
```

Comment:
It would be better to create a binding for this object in the make component itself, this function is only called once and return just a simple object.